tests/libvmi/vmi.go

```diff
@@ -185,6 +185,21 @@ func WithEvictionStrategy(evictionStrategy v1.EvictionStrategy) Option {
 	}
 }
 
+// WithHugePages configures the vmi to use hugepages
+func WithHugePages(hugePagesSize string) Option {
```

Comment:
This seems to leave no room for expanding the `Hugepages` options (without breaking the callers).
(if a new field is added to the struct)

I guess there are two options here:
- Rename to `WithHugePagesSize`.
- Pass in the `Hugepages` struct as input.

Reply:
done, thank you.